pydantic/types.py

```diff
@@ -875,6 +875,9 @@ def __repr__(self) -> str:
     def __len__(self) -> int:
         return len(self._secret_value)
 
+    def __hash__(self) -> int:
```

Comment:
I think you can move this onto `SecretField` and then both `SecretStr` and `SecretBytes` should be hashable.

You can then also add a test for `assert type(hash(SecretBytes('secret'))) is int`

Reply:
Also, you need to update your change entry as well.